rockr/views.py

```diff
@@ -44,12 +58,78 @@ def get_user_role():
     }
     return jsonify(data)
 
+@app.route('/update_user_account', methods=["POST"])
+def update_user_account():
```

Comment:
your endpoints shouldn't have the request verb in them. We can be more generic. Let's touch base on this tomorrow